pages/agent/v2.md.erb

```diff
@@ -45,7 +45,3 @@ The agent has a standard configuration file format on all systems to set meta-da
 ## Customising with Hooks
 
 The agent’s behaviour can be customized using hooks, which are just shell scripts that exist on your build machines or in each pipeline’s code repository. Hooks can be used to set up secrets as well as overriding default behavior. See the [hooks](agent/hooks) documentation for full details.
-
-## Signal Handling
```

Comment:
This seems to also be the case in `v2` of the agent, so we can probably keep this in @lox? https://github.com/buildkite/agent/blob/2-6-stable/process/process.go#L276